app.py

```diff
@@ -1,44 +1,25 @@
-import datetime
-import hashlib
-
-import discord
+from lacus.cogs.time import TimeCog
```

Comment:
Just a comment, but I was thinking of adding something like [isort](https://github.com/PyCQA/isort) which could help organize these imports.